app/jobs/check_veteran_residence_location_job.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class CheckVeteranResidenceLocationJob < CaseflowJob
+  RESIDENCE_LOCATION_PROCESS_LIMIT = if ENV["RESIDENCE_LOCATION_BATCH_SIZE"].nil?
+                                       2500
+                                     else
+                                       ENV["RESIDENCE_LOCATION_BATCH_SIZE"].to_i
+    end
+  VET_UPDATE_BATCH_PROCESS_LIMIT = ENV["VET_UPDATE_BATCH_SIZE"].nil? ? 100 : ENV["VET_UPDATE_BATCH_SIZE"].to_i
```

Comment:
```suggestion
  VET_UPDATE_BATCH_PROCESS_LIMIT = ENV.fetch("VET_UPDATE_BATCH_SIZE", 100).to_i
```
The same can apply here.